source/common/common/hash.cc

```diff
@@ -4,6 +4,17 @@
 
 namespace Envoy {
 
+uint64_t HashUtil::xxHash64(absl::Span<absl::string_view> input, uint64_t seed) {
+  if (input.empty()) {
+    seed = XXH64(input.data(), input.size(), seed);
```

Comment:
What does this do exactly? What does it mean to hash an empty list?

Reply:
Yes, just handle the corner case although we didn't use it now. see also the comment https://github.com/envoyproxy/envoy/pull/15486#discussion_r607872326

Reply:
If you just leave this case out you'll wind up return seed for an empty input vector. Is that a problem?

Reply:
no, it isn't any problem, I can leave that case out